code/solutions/package-and-deploy/scripts/deploy-cfn-template.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# From code/package-and-deploy directory run ./scripts/deploy-cfn-template.sh
+
+BUCKET_NAME="cfn-workshop-s3-s3bucket-1wo3wjmf93viq" # use your own uniquely named S3 bucket
```

Comment:
if the user has to use their own s3 bucket, we should be requiring that as input to the script and erroring if not provided. same goes for region